packages/eslint-config-ponder/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "eslint-config-ponder",
```

Comment:
I think we want to call the package `eslint-config-ponder` as shown here. I saw in a few other places it's referenced as `@ponder/eslint-config`, so I think you'll have to change those.

Reply:
jfc - that's what I get for coding before the first cup of coffee